src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -184,7 +184,7 @@ const RuntimeEventDetailsInner: FC<{
       )
     case RuntimeEventType.evmlog: {
       const { parsedEvmLogName } = parseEvmEvent(event)
-      const emittingEthAddress = `0x${Buffer.from(event.body.address, 'base64').toString('hex')}`
+      const emittingEthAddress = getEthAccountAddressFromBase64(event.body.address)
```

Comment:
missing import 
shouldn't this be normalized in API layer?

Reply:
Ah, yeah, it doesn't depend on other info than event.type. Will move